pkg/kubelet/rkt/image.go

```diff
@@ -62,13 +62,20 @@ func (r *Runtime) PullImage(image kubecontainer.ImageSpec, pullSecrets []api.Sec
 		glog.V(1).Infof("Pulling image %s without credentials", img)
 	}
 
-	// Let's update a json.
-	// TODO(yifan): Find a way to feed this to rkt.
-	if err := r.writeDockerAuthConfig(img, creds); err != nil {
+	userConfigDir, err := ioutil.TempDir("", "rktnetes-user-config-dir-")
+	if err != nil {
+		return fmt.Errorf("rkt: Cannot create a temporal user-config directory: %v", err)
+	}
+	defer os.RemoveAll(userConfigDir)
+
+	config := *r.config
+	config.UserConfigDir = userConfigDir
```

Comment:
Is this override actually needed for anything other than the `fetch` immediately following? It seems like this could be simplified to just pass in `"fetch", "--user-config=$dir"` below and not make further changes.

Is the goal for this to also eventually do more? Am I missing something?